pandas/core/dtypes/dtypes.py

```diff
@@ -655,39 +678,39 @@ class DatetimeTZDtype(PandasExtensionDtype):
     _match = re.compile(r"(datetime64|M8)\[(?P<unit>.+), (?P<tz>.+)\]")
     _cache: Dict[str_type, PandasExtensionDtype] = {}
 
-    def __init__(self, unit="ns", tz=None):
+    def __init__(self, unit: Union[str_type, "DatetimeTZDtype"] = "ns", tz=None):
```

Comment:
Is this worth an alias in pandas._typing?

Reply:
not sure, is it common to be able to pass a dtype where a string representing a unit is expected?

Reply:
That is common (almost any place accepting a dtype will also accept a string), but I don't know if there are many places that only accepts a DatetimeTZDtype

Reply:
most methods that have a unit parameter, document the unit as string type, so AFAICT, this is the only place where unit can be passed a DatetimeTZDtype instead and the unit extracted.

Reply:
maybe i've misunderstood. I assume the question was about an alias for unit. We already have an alias for Dtype.